common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -346,14 +346,19 @@ export interface ISignalMessageBase {
 	type?: string;
 
 	/**
-	 * Counts the number of signals sent by the client
+	 * Counts the number of signals sent by the sending client.
 	 */
 	clientConnectionNumber?: number;
 
 	/**
 	 * Sequence number that indicates when the signal was created in relation to the delta stream
 	 */
 	referenceSequenceNumber?: number;
+
+	/**
+	 * When specified, the signal is being targeted to the recieving client.
```

Comment:
I think we should additionally note that submit_signals_v2 feature is required.
Could say,
Client id of singular client that signal will be sent to or that has received signal.
May only be specified and expected when `IConnect.supportedFeatures['submit_signals_v2']` is `true`.